src/ioc/index.ts

```diff
@@ -1,12 +1,19 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
 export const createIoCContainer = () =>  {
   const ioc = new IoCContainer();
-  // you can register some resources right now below...
+  
+  ioc.registerClass('logger', Logger);
+  ioc.registerClass('http', HTTP);
+  ioc.registerClass('users', Users);
+  ioc.register('config', {
+    host: 'localhost',
+    port: 8080,
+  });
```

Comment:
You have to register the config that comes from the server. Please don't hardcode it.